zerver/views/documentation.py

```diff
@@ -223,11 +234,24 @@ def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
                 context["PAGE_TITLE"] = f"{article_title} | {title_base}"
             else:
                 context["PAGE_TITLE"] = title_base
-            request_notes = RequestNotes.get_notes(self.request)
-            request_notes.placeholder_open_graph_description = (
+            placeholder_open_graph_description = (
                 f"REPLACEMENT_PAGE_DESCRIPTION_{int(2**24 * random.random())}"
             )
-            context["PAGE_DESCRIPTION"] = request_notes.placeholder_open_graph_description
+            context["PAGE_DESCRIPTION"] = placeholder_open_graph_description
+
+            def update_description(response: HttpResponse) -> None:
+                if placeholder_open_graph_description.encode() not in response.content:
+                    return None
```

Comment:
I guess we don't actually have any pages in the relevant directories that set `PAGE_DESCRIPTION` manually in the HTML template. Probably it's still correct to return `None` here; I suppose we could do `nocoverage` or just invert the condition and fallthrough to `return None`; that latter option seems potentially nicer to me.